src/frontend/src/handler/create_table.rs

```diff
@@ -862,14 +864,14 @@ fn derive_connect_properties(
                     .ok_or_else(|| anyhow!("external table name must contain database prefix"))?
             }
             POSTGRES_CDC_CONNECTOR => {
-                let schema_name = connect_properties
-                    .get(SCHEMA_NAME_KEY)
-                    .ok_or_else(|| anyhow!("{} not found in source properties", SCHEMA_NAME_KEY))?;
+                let (schema_name, table_name) = external_table_name
+                    .split_once('.')
+                    .ok_or_else(|| anyhow!("external table name must contain schema prefix"))?;
```

Comment:
Shall we use the default schema (`public`) when not specified? Because not all PG users use schema.

Reply:
I think we shouldn't provide this sugar for this feature, which inherently support ingest tables from different schemas. It is better to be explicit and omit the word doesn't provide much benefit.

Reply:
At least we can make the error message more explicit to hint user to add "public" schema prefix if the user doesn't use schema in PG?